### ISSUE TITLE
Cannot see folders and files

### ISSUE THREAD

**Issue by Demir-Utku** [getcursor/cursor#2411]
After I open my project with cursor, I couldn't see any files and folders in my project. The file explorer doesn't list any files.

I got this error: "The file is not displayed in the text editor because it is a directory."

**Comment by Bryanoxx** [comment]
Same here 👍

**Comment by Stormix** [comment]
This probably related to your "explorer.fileNesting.patterns" rules, if you're using: https://github.com/antfu/vscode-file-nesting-config.git or something similar.

**Comment by Bryanoxx** [comment]
Thank you @Stormix! It was `sentry.*.config.*` within the `package.json` property of the File Nesting Config extension that was causing this error ✅

**Comment by Demir-Utku** [comment]
@Stormix yes I'm using the file nesting config plugin. What should I change in the `explorer.fileNesting.patters` config in settings?

**Comment by Newt0n** [comment]
> @Stormix yes I'm using the file nesting config plugin. What should I change in the `explorer.fileNesting.patters` config in settings?是的，我正在使用文件嵌套配置插件。我应该在设置中的`explorer.fileNesting.patters`配置中更改什么？

@Demir-Utku I'm having the same problem, deleting the updated configuration in explorer.fileNesting.patters, or turning off the explorer.fileNesting.enable optioncan fix it.

**Comment by Demir-Utku** [comment]
I updated the configuration (a small menu opened at the bottom right after opening Cursor). The issue was resolved after updating the config with a reload.

**Comment by Bonaventure-Msonoi** [comment]
for me it was the explorer, I couldn't view the files in explorer, a quick solution was to head to VIEW, then click OPEN VIEW, then click on EXPLORER and viola, your files appear. anywho, if it helps.

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
